pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -126,13 +126,28 @@ type OSDInfo struct {
 }
 
 type OrchestrationStatus struct {
-	OSDs    []OSDInfo `json:"osds"`
-	Status  string    `json:"status"`
-	Message string    `json:"message"`
+	OSDs         []OSDInfo `json:"osds"`
+	Status       string    `json:"status"`
+	PvcBackedOSD bool      `json:"pvc-backed-osd"`
+	Message      string    `json:"message"`
+}
+
+type osdProperties struct {
+	crushHostname  string //it's the hostname if its nodes based OSD provision and the pvc name in case of PVC backed OSD provision
```

Comment:
nits: 
- preferred to add the comment on the line above the property
- too many "its" :). How about rewording to clarify?

Reply:
update the verbiage to "crushHostname refers to the hostname or PVC name when the OSD is provisioned on Nodes or PVC block device, respectively."